package.json

```diff
@@ -96,6 +96,7 @@
   },
   "devDependencies": {
     "@build-tracker/cli": "^1.0.0-beta.15",
+    "@puppeteer/replay": "^0.5.0",
```

Comment:
why dev dep?

Reply:
Yeah we can make it a package dep, just wasn't sure if node users would ever need it. Better to play it safe on this one I guess.